### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by moonso**
Hi, I'm adding a [CITATION.cff](https://citation-file-format.github.io) file making it easy to cite scout with [autogenerated APA and Bibtex](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-citation-files) formated citation styles.

Please update author list or if you want you can add your orcid. It would be great for me if we can merge this now and then people can do PRs with updates later on 😄

## How to test
No tests, no code is touched

## Expected outcome
It will be easy to generated nicely formated references to scout

## Review
- [ ] code approved by

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -16,7 +16,7 @@ authors:
     given-names: Daniel
   - family-names: Rasi
     given-names: Chiara
-version: 3.7.3
```
Comment (northwestwitch) [inline CITATION.cff]: 👍🏻 I wasn't even aware of the citation file!

**Review by northwestwitch — APPROVED** [review]

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    given-names: Mats
+  - family-names: Laaksonen
+    given-names: Mikael
+  - family-names: Magnusson
+    given-names: Måns
+    orcid: "https:// https://orcid.org/0000-0002-0001-1047"
+  - family-names: Nilsson
+    given-names: Daniel
+  - family-names: Rasi
+    given-names: Chiara
```
Comment (northwestwitch) [inline CITATION.cff:18]:
```suggestion
    given-names: Chiara
    orcid: "https://orcid.org/0000-0002-7001-3988"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+authors:
+  - family-names: Andeer
+    given-names: Robin
+  - family-names: Dalberg
+    given-names: Mats
+  - family-names: Laaksonen
+    given-names: Mikael
+  - family-names: Magnusson
+    given-names: Måns
+    orcid: "https:// https://orcid.org/0000-0002-0001-1047"
```
Comment (northwestwitch) [inline CITATION.cff:14]:
```suggestion
    orcid: "https://https://orcid.org/0000-0002-0001-1047"
```

**Review by northwestwitch — APPROVED** [review]
Don't forget the changelog!

**Comment by moonso** [comment]
Can you merge it?

**Comment by northwestwitch** [comment]
> Can you merge it?

Yes, I just updated the changelog. Let's merge as soon as it passes the tests!

**Pull request merged**